core/src/main/java/nl/nn/adapterframework/http/WebServiceListener.java

```diff
@@ -208,7 +208,7 @@ public String getLogPrefix() {
 	@Override
 	public String getPhysicalDestinationName() {
 		if(StringUtils.isNotEmpty(getAddress())) {
-			return "address ["+getAddress()+"]";
+			return "address [/services/"+getAddress()+"]";
```

Comment:
Would it be possible to retrieve this string from where it is defined, instead of hard coding it?